.script/tests/detectionTemplateSchemaValidation/ValidConnectorIds.json

```diff
@@ -58,6 +60,7 @@
   "illusiveAttackManagementSystem",
   "ImpervaWAFGateway",
   "ImportedConnector",
+  "InfobloxCloudDataConnector",
```

Comment:
Where did you get these connector ids from?

Reply:
I have updated these data connector id's present in solution folder.